.evergreen/.evg.yml

```diff
@@ -550,9 +550,6 @@ functions:
 
 # Anchors
 
-hosts: &hosts
```

Comment:
Now that we also test with ubuntu, it was confusing to have this hosts anchor for just one of the variants, so I just killed it.